content/api/templates.apib

```diff
@@ -556,26 +450,99 @@ If the template delete API call succeeds, then ALL versions of the template will
 
         {}
 
-+ Response 404 (application/json)
++ Response 409 (application/json)
 
         {
             "errors": [
                 {
-                    "message": "resource not found",
-                    "code": "1600",
-                    "description": "Template does not exist"
+                    "message": "resource conflict",
+                    "code": "1602",
+                    "description": "Template is in use by msg generation"
                 }
             ]
         }
 
-+ Response 409 (application/json)
+### List all Templates [GET /templates{?draft,shared_with_subaccounts}]
+
++ Parameters
+    + draft (optional, boolean, `true`) ...If true, returns the draft template.  If false, returns the published template. When not provided, returns the most recently edited template version (draft or published).
```

Comment:
This is all singular ("If true, returns the draft template") should be plural ("If true, returns the template drafts**s**")